src/jloptions.c

```diff
@@ -188,8 +189,8 @@ static const char opts[]  =
     "                            fallbacks to the latest compatible BugReporting.jl if not. For more information, see\n"
     "                            --bug-report=help.\n\n"
 
-    " --heap-size-hint=<size>    Forces garbage collection if memory usage is higher than that value.\n"
-    "                            The memory hint might be specified in megabytes(500M) or gigabytes(1G)\n\n"
+    " --heap-size-hint=<size>    Forces garbage collection if memory usage is greater than the given number of bytes.\n"
+    "                            The memory hint might be specified in megabytes (e.g., 500M) or gigabytes (e.g., 1G)\n\n"
```

Comment:
To repeat my previous comment: The same text also occurs in `doc/man/julia.1` and with PR https://github.com/JuliaLang/julia/pull/50480 <s>hopefully soon</s> also in `doc/src/manual/command-line-interface.md` so the verbiage should be adjusted to match in those two files, too.

Reply:
Sorry, I didn't mean to blithely dismiss you — I thought I did that in [7495e55](https://github.com/JuliaLang/julia/pull/48050/commits/7495e55a4b729c57933a85018e7dfbbc0b4a4b56)!  Will address

Reply:
No worries, I assumed it got lost in the shuffle :-)